src/mgr/DaemonServer.cc

```diff
@@ -1181,6 +1257,12 @@ bool DaemonServer::_handle_command(
     return true;
   }
 
+  // Track non-admin mgr ops only
+  MessageRef mref = m.get();
+  MgrOpRequestRef op = op_tracker.create_request<MgrOpRequest, MessageRef>(mref);
```

Comment:
Only the `op_tracker` in `DaemonServer` receives the `create_request` calls.

Reply:
Yes. The op tracking in Mgr was yet to implemented. Removed `op_tracker` from Mgr.